packages/core/src/lib/GooglePlay.ts

```diff
@@ -44,7 +44,7 @@ export class GooglePlay {
    * @param serviceAccountJsonFilePath This is the service account file to communicate with the
    *   play publisher API.
    */
-  constructor(private gradleWrapper: GradleWrapper, serviceAccountJsonFilePath?: string) {
+  constructor(private gradleWrapper?: GradleWrapper, serviceAccountJsonFilePath?: string) {
```

Comment:
Why is `gradleWrapper` now optional?

Reply:
I made the `gradleWrapper` optional because in the next CL I plan on removing all the gradle changes. This is no longer needed after that change.